### PR TITLE
fix: minor typos in post

### PR DESCRIPTION
**Pull request by Ernest0day**
Just fixed up some minor typos here and there. Sorry if I am using wrong syntax or procedure. Today is my first day attempting pull requests. Thank you.

### CODE REVIEW

File: content/posts/02-exploring-redux-toolkit/index.md [hunk 1]
```diff
@@ -8,15 +8,16 @@ tags: ['redux', 'redux-toolkit', 'react']
 date: 2020-11-20
 ---
 
-I've been using Redux for managing and updating the global state in my React apps for a while. Redux makes way easier for managing the gobal state and automatically implements complex performance optimizations, so that your own component only re-renders when the data it needs has actually changed ; the major complaint about the React-Redux is **Boilerplate**. 🍽️
+I've been using Redux for managing and updating the global state in my React apps for a while. Redux makes it way easier to manage the gobal state and automatically implements complex performance optimizations, so that your own component only re-renders when the data it needs has actually changed ; the major complaint about the React-Redux is **
```
Comment (adarshaacharya) [inline content/posts/02-exploring-redux-toolkit/index.md:11]: actually, it is boilerplate not only plate. can you please fix that.

**Comment by adarshaacharya** [comment]
thanks for amazing pr. Please make one changes as i've mentioned above.

**Review by adarshaacharya — CHANGES_REQUESTED** [review]

File: content/posts/02-exploring-redux-toolkit/index.md (hunk 1 above)
Comment (Ernest0day) [reply content/posts/02-exploring-redux-toolkit/index.md:11]: oh sure np, I will fix that.

**Review by adarshaacharya — APPROVED** [review]
Thank you for fixing issue.

**Comment by Ernest0day** [comment]
You are very welcom Aadarsha! Thanks for the feedback and interesting blogs!

**Comment by adarshaacharya** [comment]
@Ernest0day thanks for fixing typos didn't realized there were so many mistakes. 😄

**Pull request merged**